src/main/java/ResponseManager.java

```diff
@@ -0,0 +1,83 @@
+public class ResponseManager {
+    private static final String LOGO =
+            "███████╗██╗   ██╗██╗  ██╗███████╗\n" +
+            "╚══███╔╝██║   ██║██║ ██╔╝██╔════╝\n" +
+            "  ███╔╝ ██║   ██║█████╔╝ █████╗  \n" +
+            " ███╔╝  ██║   ██║██╔═██╗ ██╔══╝  \n" +
+            "███████╗╚██████╔╝██║  ██╗███████╗\n" +
+            "╚══════╝ ╚═════╝ ╚═╝  ╚═╝╚══════╝\n";
+    private static final String INDENTATION_LINE =
+            "____________________________________________________________";
+    private static final String BYE_MESSAGE =
+            "Bye. Hope to see you again soon!\n";
+    private static final String TASK_ADDED_MESSAGE =
+            "Got it. I've added this task:\n";
+    private static final String MARK_ERROR_MESSAGE =
+            "please enter a valid index for mark!\n";
+    private static final String UNMARK_ERROR_MESSAGE =
+            "please enter a valid index for unmark!\n";
+    private static final String STANDARD_ERROR_MESSAGE =
+            "sorry, I don't get your words :(\n";
+    private static final String LIST_TASK_MESSAGE =
+            "Here are the tasks in your list:\n";
+    private static final String UNMARKED_MESSAGE =
+            "OK, I've marked this task as not done yet:\n";
+    private static final String MARKED_MESSAGE =
+            "Nice! I've marked this task as done:\n";
+
+    public static void indentPrint(String response) {
+        System.out.println(INDENTATION_LINE);
+        System.out.println(response + INDENTATION_LINE);
+    }
+    public static void greet() {
```

Comment:
I like how you made separate class to manage print statements and functions to handle the tasks.